src/scss/components/file/_config.scss

```diff
@@ -0,0 +1,9 @@
+/// ========== VARIABLES ========== ///
+@import "vars";
+
+/// ========== PARTIALS ========== ///
+@import "hx-file-control";
+@import "hx_drop-fence";
+@import "hx_drop-zone";
+@import "hx-file-icon";
+@import "hx-file-tile";
```

Comment:
Lines 4-9 should be in `src/scss/components/file/index.scss`.

Avoid importing files that generate CSS into `_config.scss`.

Reply:
> Rename `src/scss/components/file/_index.scss` to `index.scss` (without leading underscore) to denote that it's a potential entry point.

Where else do we do this?